src/Stripe.net/Services/OAuth/OAuthAuthorizeUrlOptions.cs

```diff
@@ -0,0 +1,76 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class OAuthAuthorizeUrlOptions : BaseOptions
+    {
+        /// <summary>
+        /// Boolean to indicate that the user should always be asked to connect, even if they're
+        /// already connected. Defaults to <c>false</c>.
+        /// </summary>
+        [JsonProperty("always_prompt")]
+        public bool? AlwaysPrompt { get; set; }
+
+        /// <summary>
+        /// The unique identifier provided to your application, found in your
+        /// <see href="https://dashboard.stripe.com/account/applications/settings">application settings</see>.
+        /// </summary>
+        [JsonProperty("client_id")]
+        public string ClientId { get; set; } = StripeConfiguration.ClientId;
+
+        /// <summary>The only option at the moment is <c>code</c>.</summary>
+        [JsonProperty("response_type")]
+        public string ResponseType { get; set; } = "code";
+
+        /// <summary>
+        /// <para>
+        /// The URL for the authorize
+        /// <see href="https://stripe.com/docs/connect/oauth-reference#get-authorize-response">response</see>
```

Comment:
Why mention this link instead of just calling it a redirect url?

Reply:
Copied straight from the docs.